storages/types.go

```diff
@@ -9,3 +15,38 @@ const (
 	SnowflakeType       = "snowflake"
 	GoogleAnalyticsType = "google_analytics"
 )
+
+type DryRunResponse struct {
```

Comment:
Seems like adapters.TableField structure. We are using it in a similar response structure in cached events response.

Reply:
You are right! I've replaced all the usages of DryRunResponse and removed the structure